src/doc/rustdoc/src/unstable-features.md

```diff
@@ -664,3 +664,22 @@ Similar to cargo `build.rustc-wrapper` option, this flag takes a `rustc` wrapper
 The first argument to the program will be the test builder program.
 
 This flag can be passed multiple times to nest wrappers.
+
+### `--extract-doctests`: outputs doctests in JSON format
+
+ * Tracking issue: [#134529](https://github.com/rust-lang/rust/issues/134529)
+
+When this flag is used, it outputs the doctests original source code alongside information
+such as:
+
+ * File where they are located.
+ * Line where they are located.
+ * Codeblock attributes (more information about this [here](./write-documentation/documentation-tests.html#attributes)).
+
+The output format is JSON.
```

Comment:
That's not enough documentation, even for an unstable feature. You need to say what the actual keys are, and what they do (particularly the subkeys of `langstr`, which are not self-explanatory). Please include a prettified example JSON document with all of the format features used.

Reply:
I'm not too sure if we want everything to be documented as docblock attributes list might get longer. Well, I'll list the current one and we can think about it later.